index.js

```diff
@@ -24,24 +61,45 @@ addTaskButton.addEventListener("click", () => {
     createdAt: new Date().toISOString(),
   });
 
+  allTasks = tasks;
+
```

Comment:
`tasks` should contain of all tasks. This leads to confusion.

Reply:
fixed